packages/angular_devkit/build_angular/src/angular-cli-files/plugins/optimize-css-webpack-plugin.ts

```diff
@@ -44,21 +45,6 @@ export class CleanCssWebpackPlugin {
 
   apply(compiler: Compiler): void {
     hook(compiler, (compilation: compilation.Compilation, chunks: compilation.Chunk[]) => {
-      const cleancss = new CleanCSS({
-        compatibility: 'ie9',
-        level: {
-          2: {
-            skipProperties: [
-              'transition', // Fixes #12408
-              'font', // Fixes #9648
```

Comment:
Does the new approach keep these issues fixed?

Reply:
Yes.